catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -238,6 +238,29 @@ public String getResponseWrapper(String responseType, Object response) {
                 EndpointUtil::firstInWinsMerge));
   }
 
+  public List<Metacard> getMetacardListByTag(String tagStr) {
+    Filter filter = filterBuilder.attribute(Core.METACARD_TAGS).is().like().text(tagStr);
+
+    return resultIterable(
+            catalogFramework,
+            new QueryRequestImpl(
+                new QueryImpl(
+                    filter,
+                    1,
+                    pageSize,
+                    new SortByImpl(Core.MODIFIED, SortOrder.DESCENDING),
+                    false,
+                    TimeUnit.SECONDS.toMillis(10)),
+                false,
+                null,
+                additionalSort(new HashMap<>(), Core.ID, SortOrder.ASCENDING)))
```

Comment:
Just curious, why do we need to sort the metacards by descending modified data then by ascending id?